ebpf/Dockerfile

```diff
@@ -0,0 +1,56 @@
+# Simple eBPF build environment
```

Comment:
Perhaps we should name the Dockerfile to be something like `Dockerfile.builder` so we need what this is for.